src/game/header/local.h

```diff
@@ -1095,4 +1115,14 @@ struct edict_s
 	monsterinfo_t monsterinfo;
 };
 
+/*
+ * Uncomment for check that exported functions declarations are same as in
+ * implementation. (-Wmissing-prototypes )
+ *
+ */
+#if 0
```

Comment:
This is a cool idea. I wonder I never had it ;)

Reply:
Thank you!